week2/extra-homework/index.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const fs = require('fs');
+const express = require('express');
+const Counters = require('./counters');
```

Comment:
for this part, the require method assumes it's argument to be a `js`, `json` or a `node` extension, and this line specifically takes the `js` extension because it's the first on the list, but to not be confused, and it's a personal style, what about renaming `counter.json` to be another name?

Reply:
Yeah there is definitely a counter naming confusion. Everywhere is full of counters 😁
Changed the json file name to counter